translations/game_modes.tex/fr.po

```diff
@@ -91,6 +97,11 @@ msgid ""
 "Further rules changes are detailed in the Campaign Mission Books.\n"
 "\n"
 msgstr ""
+"\\subsection*{Campagne}\n"
+"Un Mode Solo aux Scénarios interconnectés contre une IA ennemie.\n"
+"Pour les règles spécifiques au Mode Solo\\index{Solo Mode}, voir \\hyperlink{AIrules}{Règles IA}.\n"
```

Comment:
Index entries should be translated. Is this how you want it to be in the Index page?